src/components/Card.js

```diff
@@ -65,6 +101,11 @@ class Card {
     this._cardElement.querySelector(".card__title").textContent = this._name; // Set the title of the card
     this._cardElement.querySelector(".card__image").src = this._link; // Set the image source of the card
     this._cardElement.querySelector(".card__image").alt = this._name; // Set the alt text of the card image
```

Comment:
**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `generateCard`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.